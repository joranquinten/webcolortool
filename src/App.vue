<template>
  <v-app dark>
    <v-toolbar app>
      <v-toolbar-title class="headline text-uppercase">
        <span>WebColorTool</span>
      </v-toolbar-title>
      <v-spacer></v-spacer>
      <v-btn
        flat
        href="https://github.com/joranquinten/webcolortool"
        target="_blank"
      >
        <span class="mr-2">github</span>
      </v-btn>
    </v-toolbar>

    <v-content>
      <color-tool />
    </v-content>
    <v-footer dark height="auto">
      <v-card-text class="white--text">
        &copy; 2019 â€”
        <a href="https://joranquinten.nl/">Joran Quinten</a>
      </v-card-text>
    </v-footer>
  </v-app>
</template>

<script>
import ColorTool from "./components/ColorTool";

export default {
  name: "App",
  components: {
    ColorTool
  }
};
</script>

<style lang="scss">
*::-moz-selection {
  color: #44475a;
  background-color: #f1fa8c;
}
*::selection {
  color: #44475a;
  background-color: #f1fa8c;
}

.clickable {
  cursor: pointer;
}
</style>
