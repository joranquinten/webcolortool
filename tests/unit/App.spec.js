import { mount } from "@vue/test-utils";
import App from "../../src/App";
import Vue from "vue";
import Vuetify from "vuetify";
Vue.use(Vuetify);

describe("<App />", () => {
  it("should render without crashing", () => {
    const wrapper = mount(App, {
      stubs: {
        ColorTool: "<div class='color-tool'></div>"
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
