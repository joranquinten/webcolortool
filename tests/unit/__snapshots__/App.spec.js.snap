// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render without crashing 1`] = `
<div data-app="true" class="application theme--dark" id="app">
  <div class="application--wrap">
    <nav class="v-toolbar v-toolbar--fixed theme--dark" style="margin-top: 0px; padding-right: 0px; padding-left: 0px; transform: translateY(0px);">
      <div class="v-toolbar__content" style="height: 64px;">
        <div class="v-toolbar__title headline text-uppercase"><span>WebColorTool</span></div>
        <div class="spacer"></div> <a href="https://github.com/joranquinten/webcolortool" target="_blank" class="v-btn v-btn--flat theme--dark">
          <div class="v-btn__content"><span class="mr-2">github</span></div>
        </a>
      </div>
    </nav>
    <main class="v-content" style="padding: 64px 0px 0px 0px;">
      <div class="v-content__wrap">
        <div class="color-tool"></div>
      </div>
    </main>
    <footer class="v-footer theme--dark" style="height: auto;">
      <div class="v-card__text white--text">
        © 2019 —
        <a href="https://joranquinten.nl/">Joran Quinten</a></div>
    </footer>
  </div>
</div>
`;
